### ISSUE TITLE
Failed to fetch

### ISSUE THREAD

**Issue by iamaandre** [helabenkhalfallah/react-apollo-redux-graphql#1]
By following the steps I could run the code but when I change the route by /users, I am getting notified as Network Error. 

I believe the server is not running. 

can you help me to fix this issue ?

**Comment by helabenkhalfallah** [comment]
Yes sure, can you send me your github repository to take a look. Thanks.

**Comment by helabenkhalfallah** [comment]
I think you should start the server, here's the server code : 
https://github.com/helabenkhalfallah/node-graphql-api
